x-pack/plugins/enterprise_search/public/applications/app_search/components/relevance_tuning/components/empty_state.test.tsx

```diff
@@ -21,7 +23,7 @@ describe('EmptyState', () => {
 
     expect(wrapper.find('h2').text()).toEqual('Add documents to tune relevance');
     expect(wrapper.find(EuiButton).prop('href')).toEqual(
-      expect.stringContaining('/relevance-tuning-guide.html')
+      expect.stringContaining(`${docLinks.appSearchRelevance}`)
```

Comment:
```suggestion
      expect.stringContaining(docLinks.appSearchRelevance)
```
No need for a template literal here